bower.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-copy-to-clipboard",
+  "version": "1.0.1",
+  "homepage": "https://github.com/nkbt/react-copy-to-clipboard",
+  "authors": [
+    "Nik Butenko <nik@butenko.me> (http://butenko.me/)"
+  ],
+  "description": "Copy-to-clipboard React component",
+  "main": "lib/index.js",
```

Comment:
Hm, I have some ideas how to automate this stuff with CI without a need to push source into a repo